datacube/index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Minimum Snow and Ice</title>
```

Comment:
Put `<title>` after both `<meta>` elements, for consistency with the other documents.